<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Initialize</class>
 <widget class="QWidget" name="Initialize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>玩家人数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="playerNumberCombo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>初始金钱：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="initialMoneyEdit">
           <property name="text">
            <string>10000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>玩家1：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="player1Combo">
               <item>
                <property name="text">
                 <string>曹操</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>刘备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>孙权</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>选择武将：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="general1Table">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>玩家2：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="player2Combo">
               <item>
                <property name="text">
                 <string>刘备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>曹操</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>孙权</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>选择武将：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="general2Table">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>29</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>玩家3：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="player3Combo">
               <item>
                <property name="text">
                 <string>孙权</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>刘备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>曹操</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>选择武将：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="general3Table">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>开始游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
